test/unit/model/virtualbox-test.js

```diff
@@ -240,10 +241,17 @@ describe('Virtualbox installer', function() {
       }
     });
 
+    it('should able to check file were extracted to temp', function(done) {
```

Comment:
Did you mean to name it as 'should run virtualbox installer executable to extract msi installer into temp folder'?